chapters/ch04.1-refactoring-the-code.md

```diff
@@ -39,9 +39,8 @@ lib/
 ├── config/
 │     ├── rolling-config.js
 │     ├── log-config.js
-├── index.js (entry point)
-├── package.json
-└── logger.js
```

Comment:
We do need the `logger.js` file here.

Reply:
**First I would like to thank you so much for your incredible work on this book <3**




Correct me if I am wrong, the following screenshot has no` logger.js` in the root but the structure shows that it is in the root.

![image](https://github.com/ishtms/learn-nodejs-hard-way/assets/62201748/7d993a69-ef1a-46b5-b4ef-883931e9674a)

**This following screenshot shows that it is in the `lib/` directory not the root**   

![image](https://github.com/ishtms/learn-nodejs-hard-way/assets/62201748/ad008042-d7ce-4413-946f-d80341c6ca47)

Reply:
Ah! you're totally correct, my bad. That in-fact is a wrong directory structure that I've posted in the chapter 4.1
Thanks for clearing it up @mki98, cheers!

Reply:
You're welcome! I'm glad I could help ❤️